### PR TITLE
Prepared UI for manually adding authors links.

### PR DESCRIPTION
**Pull request by ookmm**

### CODE REVIEW

File: social/app/templates/posts/post_form2.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+{% block content %}
+    <div class="panel panel-default">
+        <div class="panel-body">
+            <form action="" method="post" enctype="multipart/form-data">
+                {% csrf_token %}
+
+                {{ form.non_field_errors }}
+
+                <div class="form-group">
```
Comment (davidyee) [inline social/app/templates/posts/post_form2.html:17]: Can these be in a for loop to reduce redundant code?

File: social/static/js/site.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+            $(".sd-comment-text-small").shorten({showChars: 10});
+
+        });
+    }
+
+    shortenText();
+
+    function addAuthorFields() {
+        var maxFields = 10; // Max number of authors that can be added
```
Comment (davidyee) [inline social/static/js/site.js:19]: Could be nice to remove the [+] symbol when the `count == maxFields`.

File: social/app/templates/posts/includes/post_header.html [hunk 3]
```diff
@@ -1,4 +1,9 @@
-<h4>{{ post.author.displayName }}</h4>
+<h4 class="">
```
Comment (davidyee) [inline social/app/templates/posts/includes/post_header.html:1]: Probably not necessary to declare empty classes.

**Review by davidyee — APPROVED** [review]

File: social/app/templates/posts/includes/post_header.html (hunk 3 above)
Comment (ookmm) [reply social/app/templates/posts/includes/post_header.html:1]: I meant to add a class there.

File: social/static/js/site.js (hunk 2 above)
Comment (ookmm) [reply social/static/js/site.js:19]: Can check that.

File: social/app/templates/posts/post_form2.html (hunk 1 above)
Comment (ookmm) [reply social/app/templates/posts/post_form2.html:17]: Will take a look.

**Pull request merged**